book/src/sprites/modify_the_texture.md

```diff
@@ -0,0 +1,61 @@
+# Modify The `Texture`
+
+The colours of the sprite will show up exactly as in the source file,
+but sometimes being able to slightly modify the overall colour
+is useful - for instance, colouring an angry enemy more red, or
+making a frozen enemy blue. Amethyst has a [`Component`][doc_component] called
+[`Tint`][doc_tint] to do this.
+
+To use [`Tint`][doc_tint], register [`Tint`][doc_tint] as a new
+[`Component`][doc_component] with the world and build it as part of the entity.
+[`Tint`][doc_tint] will multiply the colour values of the sprite by its
+own values, so a [`Tint`][doc_tint] with a white colour will have no
+effect on the sprite.
+
+```rust,edition2018,no_run,noplaypen
+# extern crate amethyst;
+use amethyst::assets::{AssetStorage, Handle, Loader};
+use amethyst::core::Transform;
+use amethyst::prelude::*;
+use amethyst::renderer::{ImageFormat, Texture};
+use amethsyt::renderer::resources::Tint;
+use amethyst::renderer::palette::rgb::Srgba;
+
+fn init_image(world: &mut World, texture_handle: &Handle<Texture>) {
+    use amethyst::core::math::RealField;
+
+    // Add a transform component to give the image a position
+    let mut transform = Transform::default();
+    transform.set_translation_x(0.0);
+    transform.set_translation_y(0.0);
+    
+    // Flip horizontally
+    transform.set_rotation_y_axis(f32::pi());
+
+    // Colour white to show as normal, then update later
+    let colour = Tint(Srgba::new(1.0, 1.0, 1.0, 1.0));
+
+    world
+        .create_entity()
+        .with(transform)
+        .with(colour)
+        .with(texture_handle.clone()) // Use the texture handle as a component
```

Comment:
Why `.clone()` the handle? Also I don't think the comment is needed.

Reply:
The code sample was copied from the Display The Texture page, so the comment was left in - I'm happy to remove it, but surely the clone is either necessary in both places and should be kept, or should be removed from both places?

Reply:
It doesn't seem necessary in either place. I think you can remove it as well as the `&` in the function signature and where the function is called. (Unless there's a reason for not just using the first `Handle` that I'm overlooking.)

Reply:
I gave it a quick test and you're right - modified in commit.